scheduler/src/cook/mesos/optimizer.clj

```diff
@@ -19,122 +19,251 @@
             [clj-http.client :as http]
             [clj-time.core :as t]
             [clj-time.coerce :as tc]
-            [clj-time.periodic :as periodic]
-            [clojure.core.async :as async]
             [clojure.tools.logging :as log]
-            [cook.util :refer [lazy-load-var PosNum PosInt NonNegInt]]
+            [cook.util :refer [lazy-load-var NonNegInt PosNum PosInt]]
```

Comment:
🙄

You didn't move `lazy-load-var` to the end of the list???
`(sort ["lazy" "Non"])` → `("Non" "lazy")`

And you didn't move `cook.util` below `cook.mesos.util`??????